spec/victor-cli/commands/init_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "spec_helper"
+
+describe "victor init" do
+  subject { CommandLine.router }
+
+  context "without arguments" do
+    it "shows short usage" do
+      expect { subject.run %w[init] }
+        .to output_fixture("cli/init/usage")
+    end
+  end
+
+  context "with --help" do
+    it "shows long usage" do
+      expect { subject.run %w[init --help] }
+        .to output_fixture("cli/init/help")
+    end
+  end
+
+  context "with RUBY_FILE" do
```

Comment:
Perhaps it is good to test when the `RUBY_FILE` already exists as well?

Reply:
Yes. Not only test, but also avoid overwriting existing files, since there is more chance to cause harm than good here if we don't protect existing files. 

Done + added a spec for it.